website/docs/r/neptune_cluster.html.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: "aws"
+page_title: "AWS: aws_neptune_cluster"
+sidebar_current: "docs-aws-resource-neptune-cluster"
+description: |-
+  Provides an Neptune Cluster Resource
+---
+
+# aws_neptune_cluster
+
+Provides an Neptune Cluster Resource. A Cluster Resource defines attributes that are
+applied to the entire cluster of Neptune Cluster Instances.
+
+Changes to a Neptune Cluster can occur when you manually change a
+parameter, such as `backup_retention_period`, and are reflected in the next maintenance
+window. Because of this, Terraform may report a difference in its planning
+phase because a modification has not yet taken place. You can use the
+`apply_immediately` flag to instruct the service to apply the change immediately
+(see documentation below).
+
+## Example Usage
+
+```hcl
+resource "aws_neptune_cluster" "default" {
+  cluster_identifier      = "neptune-cluster-demo"
+  engine                  = "neptune"
+  backup_retention_period = 5
+  preferred_backup_window = "07:00-09:00"
+  skip_final_snapshot = "true"
+  iam_database_authentication_enabled = "true"
+  apply_immediately = "true"
+}
+```
+
+~> **Note:** AWS Neptune does not support user name/password–based access control.
+See the AWS [Docs](https://docs.aws.amazon.com/neptune/latest/userguide/limits.html) for more information.
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `cluster_identifier` - (Optional, Forces new resources) The cluster identifier. If omitted, Terraform will assign a random, unique identifier.
```

Comment:
The arguments should be alphabetically sorted, they are fairly hard to follow with the current ordering.